### PR TITLE
added clarity

### PR DESCRIPTION
**Pull request by ConsciousClay**
This passage was confusing to me. I made some changes that I think reflect the intended meaning. Also, I'm not sure what the clause "**as the padding and border are added to the width used for the content box**" is supposed to mean.

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
I guess, knowing about [box-sizing](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) is relevant here.

File: files/en-us/learn/css/building_blocks/the_box_model/index.html [hunk 1]
```diff
@@ -130,7 +130,7 @@ <h3 id="The_standard_CSS_box_model">The standard CSS box model</h3>
 }
 </pre>
 
-<p>The space taken up by our box using the standard box model will actually be 410px (350 + 25 + 25 + 5 + 5), and the height 210px (150 + 25 + 25 + 5 + 5), as the padding and border are added to the width used for the content box.</p>
+<p>Using the standard box model, the width of our box will be 410px (350 + 25 + 25 + 5 + 5), and the height 210px (150 + 25 + 25 + 5 + 5).</p>
```
Comment (hamishwillee) [inline files/en-us/learn/css/building_blocks/the_box_model/index.html:133]: This feels like a loss of information - it is relevant to users that "the padding and border are added to the width used for the content box"

Comment (rachelandrew) [reply files/en-us/learn/css/building_blocks/the_box_model/index.html:133]: agreed, I'm going to close this PR given that two of us had the same thoughts!

**Comment by rachelandrew** [comment]
This suggestion removes useful information, as confirmed by two reviewers so I'm closing it.

**Pull request closed without merging**